### PR TITLE
Testnet checkpoint

### PR DESCRIPTION
**Pull request by kushti**
In this PR:

* `213.239.193.208:9022` added to testnet's seed peers list
* checkpoint added to testnet settings. As some v.3 blocks in the PaiNet are violating monotonic creation height rule (due to 5.0 being activated before the monotonic introduced), this checkpoint is mandatory

### CODE REVIEW

**Review by greenhat — APPROVED** [review]

File: src/main/resources/mainnet.conf [hunk 1]
```diff
@@ -57,9 +57,9 @@ ergo {
 
     # Optional and individual checkpoint.
     # Before the height given (including it) validation of scripts is missed.
-    # This improving perfomance and memory usage during initial bootstrapping.
```
Comment (pragmaxim) [inline src/main/resources/mainnet.conf:60]: The start of the checkpoint doc is very confusing, meaning of checkpoint should be explained here

File: src/main/resources/testnet.conf [hunk 2]
```diff
@@ -91,6 +96,7 @@ scorex {
     nodeName = "ergo-testnet-5.0"
     nodeName = ${?NODENAME}
     knownPeers = [
+      "213.239.193.208:9022",
       "147.182.137.218:9022",
```
Comment (mgpai22) [inline src/main/resources/testnet.conf:102]: Add the following nodes:
- 138.2.79.187:9052
- 192.234.196.165:9052

**Review by mgpai22 — CHANGES_REQUESTED** [review]
Add the following nodes:

- 138.2.79.187:9052
- 192.234.196.165:9052

File: src/main/resources/testnet.conf (hunk 2 above)
Comment (kushti) [reply src/main/resources/testnet.conf:102]: :9022 you mean ?

Comment (reqlez) [reply src/main/resources/testnet.conf:102]: Yes he meant 9022 those are mine. I just realized that 138.2.79.187:9022 is NOT a real static, please replace that one with 168.138.185.215:9022 instead, that one will be static.

So add:  168.138.185.215:9022 + 192.234.196.165:9022

**Comment by reqlez** [comment]
> Add the following nodes:
> 
>     * 138.2.79.187:9052
> 
>     * 192.234.196.165:9052

168.138.185.215:9022 + 192.234.196.165:9022  ( please don't add .187 it's not real static )

File: src/main/resources/testnet.conf (hunk 2 above)
Comment (kushti) [reply src/main/resources/testnet.conf:104]: ok, done

File: src/main/resources/mainnet.conf (hunk 1 above)
Comment (kushti) [reply src/main/resources/mainnet.conf:60]: Improved this comment a bit, please check

Comment (pragmaxim) [reply src/main/resources/mainnet.conf:60]: This hit the spot, thanks :+1:

**Review by pragmaxim — APPROVED** [review]

**Review by reqlez — APPROVED** [review]

**Pull request merged**